open_academy/models/open_academy_course.py

```diff
@@ -9,3 +9,16 @@ class OpenAcademyCourse(models.Model):
     description = fields.Text()
     responsible_id = fields.Many2one("res.users")
     sessions_ids = fields.One2many("open.academy.session", "course_id")
+
+    _sql_constraints = [
+        ("title_description_diferents", "check(title!=description)",
+        "the title and the description must be diferents"),("title_unique", "unique(title)",
+        "this course title is already taken "),
+    ]
+
+    def copy(self, default=None):
+        if default == None:
+            default = {}
+        default["title"] = self.title + " copy"
+        rtn = super(OpenAcademyCourse, self).copy(default=default)
```

Comment:
The standard to name a variable for return a call of a super method is `res` or `result` but this is not necessary you could do
 ```suggestion
        return super(OpenAcademyCourse, self).copy(default=default)
```
Also, did you use precommit on this change?

Super with parameters is no longer used, please run precommit and change this.